Onboarding Documents/Welcome to Tidepool.md

```diff
@@ -42,9 +42,9 @@ We want you to stick around. Your successes here will be your team’s successes
 
 **Tidepool isn’t a normal medical software company.** Medical device and software companies are traditionally closed and inaccessible. At Tidepool, we're working to make software that helps people living with diabetes, and to do so in a way that is open and free (in every sense of the word).
 
-We're a non-profit, which simply put means we aren't in this for the money. We're doing this because We're open source, because we want the technology we create to be freely available and to catalyze even further innovation. (You can read more about why we are open source and non-profit in this blog post.)
+We're a non-profit, which simply put means we aren't in this for the money. We're doing this because we're open source, because we want the technology we create to be freely available and to catalyze even further innovation. (You can read more about why we are open source and non-profit in this [blog post](https://tidepool.org/why-tidepool-is-open-and-non-profit/).)
```

Comment:
We're => we're (in the middle of a sentence) and added the blog post I think you wanted to reference.